arbitrum-docs/partials/_contribute-docs-partial.md

```diff
@@ -0,0 +1,258 @@
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+:::tip Submitting a docs PR?
+
+**Please review this guidance before issuing pull requests** into the [Arbitrum docs repository](https://developer.arbitrum.io/). This guidance helps us **streamline** and **scale** our efforts. Thank you!
+
+:::
+
+<PublicPreviewBannerPartial />
+
+The [`developer.arbitrum.io`](https://developer.arbitrum.io/) docs portal is the **single source of truth** for documentation that supports Offchain's product portfolio. This includes documentation for:
```

Comment:
```suggestion
The [`docs.arbitrum.io`](https://docs.arbitrum.io/) docs portal is the **single source of truth** for documentation that supports Offchain Labs' product portfolio. This includes documentation for:
```

Subdomain has changed since yesterday I think 😄 
Also, noticed that we use "Offchain Labs" throughout the whole document except here.